5-browser-extension/translations/README.tr.md

```diff
@@ -0,0 +1,30 @@
+# Bir tarayıcı uzantısı oluşturma
+
+Tarayıcı uzantıları oluşturmak, farklı türde bir web varlığı oluştururken uygulamalarınızın performansı hakkında düşünmenin eğlenceli ve ilginç bir yoludur. Bu modül, tarayıcıların nasıl çalıştığı ve bir tarayıcı uzantısının nasıl uygulanacağı, nasıl form oluşturulacağı, bir API'nin nasıl çağrılacağı ve yerel depolamanın nasıl kullanılacağı ve web sitenizin performansının nasıl ölçüleceği ve ayrıca nasıl iyileştirileceği ile ilgili dersleri içerir.
+
+Edge, Chrome ve Firefox'ta çalışan bir tarayıcı uzantısı oluşturacaksınız. Çok özel bir görev için tasarlanmış mini bir web sitesine benzeyen bu uzantı, belirli bir bölgenin elektrik kullanımı ve karbon yoğunluğu için [C02 sinyal API'sini](https://www.co2signal.com) kontrol eder.
+
+Edge, Chrome ve Firefox'ta çalışan bir tarayıcı uzantısı oluşturacaksınız. Çok özel bir göreve uyarlanmış mini bir web sitesine benzeyen bu uzantı, belirli bir bölgenin elektrik tüketimi ve karbon yoğunluğu için [C02 Signal API'sini](https://www.co2signal.com) kontrol eder ve bölgenin karbon ayak izi hakkında bir yazı gösterir.
+
+### Konular
+
+1. [Tarayıcı hakkında](about-browsers/README.md)
```

Comment:
Since this readme will be within the "translations" folder, these 3 links will need to be prefixed with "../" to navigate back to the main folder. 

Also, these 3 links are missing the number in their URL path. 

For example, this link should be `[Tarayıcı hakkında](../1-about-browsers/README.md)`